spec/lib/versioning/revision_updater/image_spec.rb

```diff
@@ -61,13 +61,27 @@
       expect(updater.removed_lead_image?).to be_falsey
     end
 
+    it "preserves the lead image if its the current image revision" do
```

Comment:
We don't normally combine tests for multiple methods in the same `describe` block - we want to be clear about which method we're testing. I think we can have two groups of test here: one for `#update_image` and a separate group for `#update_lead_image`.

Reply:
Sure